packages/build/src/core/constants.js

```diff
@@ -57,6 +59,10 @@ const getConstants = async function({
      * The Netlify API access token
      */
     NETLIFY_API_TOKEN: token,
+    /**
+     * The path to the buildbot server socket
+     */
+    BUILDBOT_SERVER_SOCKET: triggerDeployWithBuildbotServer && buildbotServerSocket,
```

Comment:
Should `BUILDBOT_SERVER_SOCKET` be always available to core plugins even when `triggerDeployWithBuildbotServer` is `false`? This might be useful if we use the buildbot TCP socket for other purposes.